Utilities/Mergers/src/mergersTopologyExample.cxx

```diff
@@ -39,142 +41,123 @@ void customize(std::vector<CompletionPolicy>& policies)
 #include <memory>
 #include <random>
 
-using namespace std::chrono;
+using SubSpecificationType = o2::header::DataHeader::SubSpecificationType;
 
-// clang-format off
```

Comment:
I am glad to see that now clang-format behaves better with lambdas in structures than it used to :)

Reply:
it behaved weirdly because of c-style casts of lambdas, when I changed that to static_casts it started to behave correctly